internal/cucumber/config.go

```diff
@@ -231,7 +232,7 @@ func shardSuites(rootDir string, suites []Suite, ccy int) ([]Suite, error) {
 	var shardedSuites []Suite
 
 	for _, s := range suites {
-		if s.Shard != "spec" && s.Shard != "concurrency" {
+		if s.Shard != "spec" && s.Shard != "concurrency" && s.Shard != "scenario" {
```

Comment:
This line is just checking if sharding is disabled, right? 🤔 

Wouldn't this do the trick then?
```suggestion
		if s.Shard == "" {
```

Reply:
👍